### ISSUE TITLE
Doesn't work style at only production environment. 

### ISSUE THREAD

**Issue by edmitton** [splitrb/split#649]
## What
Doesn't work stylesheet at only production environment. 
Local is nope.
How can I get it to work?
help me~.
![スクリーンショット 2020-12-01 19 19 08](https://user-images.githubusercontent.com/42757908/100729545-3d710100-340c-11eb-95ed-8c8c2433bfe9.png)
![スクリーンショット 2020-12-01 19 19 24](https://user-images.githubusercontent.com/42757908/100729560-41048800-340c-11eb-8c3d-8cc7aa2f3260.png)

**Comment by davefaliskie** [comment]
@ecaron I'm having the same issue, did you find a fix for this?

**Comment by edmilton** [comment]
Hey @davefaliskie ! I think there is a typo here. 
Would you like to mention @edmitton ? (just a "T" in place of the "L") :)

**Comment by davefaliskie** [comment]
> Hey @davefaliskie ! I think there is a typo here.
> Would you like to mention @edmitton ? (just a "T" in place of the "L") :)

@edmilton yes sorry, @edmitton for what it's worth the post here solved the issue for me by moving the assets to /public https://github.com/splitrb/split/issues/496
